### PR TITLE
Update role vars and role variable description

### LINKED ISSUE #50: Update role for removal of Python 2 from Debian
# 💡 Summary #

This Ansible role should be updated for the impending removal of Python 2 from all future releases of Debian Linux.

## Motivation and context ##

The `python2` and `python2-minimal` packages were removed from Debian Bookworm just recently per the [`python2.7` news tracker](https://tracker.debian.org/news/1396456/python27-removed-from-testing/), which necessitated some fixes in #49.  We believe that this is just the first wave of finally removing Python 2 from Debian completely; the [`python2-rm` tracker](https://release.debian.org/transitions/html/python2-rm.html) is now returning a 404 error, so it appears that they are finished.

If this is all correct then, after the eventual release of Debian Bookworm, `Debian_bookworm.yml` should become `Debian.yml` and the current `Debian.yml` should become `Debian_bullseye.yml`.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request updates role variables and a role variable description:
- Edit README.md to make it clear what platforms still support Python 2.
- Ensure that all platforms install the correct packages, and can be expected to continue to do so as new platforms are released.

## 💭 Motivation and context ##

I realized that the role variable description in this role needed to be updated when working on cisagov/ansible-role-pip#61.  While I was here I figured I would do some housekeeping.

Resolves #50.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by dv4harr10 — APPROVED** [review]

File: vars/Debian.yml
Comment (mcdonnnj) [inline vars/Debian.yml:1]: Would you update the language in this variables file to match the language in `Amazon.yml` re: Python 2 support? Alternatively adjusting `Amazon.yml`; I would just like them to be consistent.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ One request for your consideration.

File: vars/Debian.yml
Comment (jsf9k) [reply vars/Debian.yml:1]: Please see commit 203ba49.

**Pull request merged**